Build-Module.ps1

```diff
@@ -100,8 +108,8 @@ if ($ipmo.IsPresent -or $runTests.IsPresent) {
    if ((Get-Module VSTeam)) {
       Remove-Module VSTeam
    }
-
-   Import-Module "$output/VSTeam.psd1" -Force
+   $env:testing = $true

```

Comment:
I don't want any testing flags anywhere. The code should be written not to require this. Tests should be able to mock their way around any need for a testing flag.

Reply:
That's a hangover which should have been removed.  Either I grabbed the wrong version of build.ps1 after sebastian said he didn't understand why the build was breaking when I left it to you guys to take the "build-classes to PSM1" change or not, or I never took it out. Nothing should be referring to it any more.  
But  from the last few PRs I've put in you want to ignore my build.ps1 , ignore my change to the psm1 file and undo the the change to where _classes.json builds to.  (I'm going to keep two versions, one which works for my development and one to put in PRs for you :-) )